src/main/g8/src/main/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+$if(add_sql_orm.truthy)$db.default {
```

Comment:
As this file will only exist if there is a add_sql_orm maybe the condition should be in the name